frontend/src/v5/ui/components/shared/linkCard/teamspaceCard/teamspaceCard.component.tsx

```diff
@@ -52,11 +56,7 @@ export const TeamspaceCard = ({ teamspaceName, imageURL, className }: ITeamspace
 					: <FormattedMessage id="teamspaceCard.sharedWithMe" defaultMessage="Shared with me" />
 			}
 		>
-			{
-				isPersonalTeamspace
-					? <MyTeamspaceImage user={user} isButton={false} />
-					: <OtherTeamspaceImage src={imgSrc} onError={onImageError} />
-			}
+			<TeamspaceImage user={user} isButton={false} />
```

Comment:
Talking about the differences between TeamSpace's images and users avatar images a bit with Carmen it became apparent that we need to use its own component for the TeamSpace images.  
As we talked earlier (I'm writing it here for the record), we could create a coverImageComponent. this componet would have the style of the cover and deal with a imgsrc prop and a defaultImageSrc in case the imgSrc is not found (the name of these props can change).